Doc/whatsnew/3.14.rst

```diff
@@ -207,6 +207,11 @@ Others
   It had previously raised a :exc:`DeprecationWarning` since Python 3.9. (Contributed
   by Jelle Zijlstra in :gh:`118767`.)
 
+* The :class:`int` built-in will no longer delegate to
```

Comment:
I think that it should be either ```The :func:`int` built-in``` or ```The :class:`int` constructor```. Preferably the former, since we will get a link to the constructor.

Reply:
Done in https://github.com/python/cpython/pull/119743/commits/7492da8cb64922147227858e9ef5cd120632a8ba